### ISSUE TITLE
Cannot preview/profile separate lexer/parser

### ISSUE THREAD

**Issue by pavlo** [antlr/intellij-plugin-v4#160]
Hi, I'm using IntelliJ IDEA 14.0.3 CE and ANTLR v4 grammar plugin of version 1.6. I copied XML parser and lexer from https://github.com/antlr/grammars-v4/tree/master/xml and was going to give it a try (learning how to do combined grammars properly).

The issue is that the 'Preview -> Parse tree' view does not render the result tree, instead it has this label: _Issues with parser and/or lexer grammar(s) prevent to preview_

I assume it is bug because the parser and the lexer have been taken from official grammars repository and it seems I've set up the plugin and project properly locally. 

Attaching the whole Intellij screen in hope it provides you with more information to be able to identify the reason of the issue. Please let me know if I can provide more details.

![xmlparser - antlrparsers - antlrparsers - _projects_govwhiz_antlrparsers](https://cloud.githubusercontent.com/assets/196921/5987418/3769e87e-a931-11e4-832f-06996a3ae2f9.png)

**Comment by pavlo** [comment]
Explored it a little further, here's what is at the bottom of plugin log:

_INFO -  ANTLR ANTLRv4PluginController - parseText can't parse: missing lexer or parser no Grammar object for /Users/pa/Projects/GovWhiz/AntlrParsers/XMLParser.g4_

After that I tried to put the g4 files into various directories directories, marked the _gen_ folder as sources etc with no luck. What else I could try?

More logs can be seen here: https://gist.github.com/pavlo/0a256bfdcc91a030ae7f

**Comment by pavlo** [comment]
sorry, closed accidentally, reopening.

**Comment by parrt** [comment]
hiya. something to do with separate lexers/parsers. can't remember if i allow that yet.

**Comment by parrt** [comment]
Note to self: Changes to lexer should update parser. Parser should see XLexer for X.g4 or XParser.g4 during underlining and preview.

**Comment by javadestroyer** [comment]
I'm facing this problem now after I had to split the parser and the lexer because I want to use the modes option. I tried deleting the files and regenerating etc. and nothing seems to work. Any ideas on how to make this work? Many thanks :)
<img width="1552" alt="Screen Shot 2024-01-12 at 06 05 43" src="https://github.com/antlr/intellij-plugin-v4/assets/45371711/6dc5495b-4715-40fe-97ee-9fb5bdee8894">
